sound/soc/codecs/pcm512x.c

```diff
@@ -1627,10 +1627,13 @@ static struct gpiod_lookup_table pcm512x_gpios_table = {
 	},
 };
 
+static const char *sclk_name = "sclk";
```

Comment:
Don't think it has to be static, it gets duplicated in `devm_clk_get()`

Reply:
static here just means local - not global variable. Doesn't hurt, does it?